edurange_refactored/react/pages/home/dashboard/scenarios/src/guide/controls/FootControls.js

```diff
@@ -21,8 +21,8 @@ function FootControls({ guideContent, updatePane, paneSide }) {
   if ((!meta)) { return (<>Scenario not found</>); }
   // GUARD
   
-  const SSH_key = `${guideContent.unique_scenario_name}_nat`
-  const SSH_IP = guideContent.SSH_IP[SSH_key]
+  const SSH_key = `${guideContent.unique_scenario_name}_StartingLine`
```

Comment:
double check

Reply:
Dependent on NAT/Gateway situation outcome... (this IS an issue but should come after that is resolved)

Reply:
actually this is frontend.  maybe okay.  will triple check.

Reply:
Removed; was an artifact of old code and unused.